### PR TITLE
Image file to data array , data array to image

### PR DESCRIPTION
**Pull request by mwharris1313**

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -1,35 +1,23 @@
...
+  var dvv = require('./dvv.js');
+
+  dvv.config({
+    staticPath: '/../client',
+    timeout: 25000,
+    data: dataArray,
+    width: width,
+    height: height,
+    func: 'math.inv',
+    clock: true
```
Comment (kevhuang) [inline server/server.js]: Can you add a comma at the end of this line?

File: server/utils/image.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+    callback(dataArray, parsedData.width, parsedData.height);
+
+  });
+
+
+
+};
+
+imageUtils.createJpegFileFromDataArray = function(filename, dataArray, width, height, callback){
```
Comment (jyeg) [inline server/utils/image.js:66]: Mike would it be possible to add some comments as to what is happening at each major point in this code?

File: server/server.js [hunk 3]
```diff
@@ -1,35 +1,23 @@
...
- clock: true,
- callback: function(results) { return results; }
-});
+  dvv.start();
+
+};
 
-dvv.start();
\ No newline at end of file
+imageUtils.createDataArrayFromJpegFile('server/rgb.jpg', init);
```
Comment (kevhuang) [inline server/server.js]: @mwharris333 looks like the path to the image is wrong. Try using `imageUtils.createDataArrayFromJpegFile(__dirname + '/assets/rgb.jpg', init);`

**Comment by kevhuang** [comment]
Looks good @mwharris333

**Comment by jyeg** [comment]
go for it

**Comment by onelovelyname** [comment]
I like the universe photos

**Pull request merged**